src/vrep_plugin.cpp

```diff
@@ -782,6 +786,72 @@ static mc_control::MCGlobalController & gc()
   return *gc_ptr;
 }
 
+static void setup_datastore()
+{
+  auto & ctl = gc_ptr->controller();
+  auto add_datastore_entries = [](mc_control::MCController & ctl) {
+    auto & ds = ctl.datastore();
+    /**
+     * @brief Controllers can check whether they are running in CoppeliaSim if this flag is present.
+     */
+    ds.make<bool>("CoppeliaSim", true);
+    /**
+     * @brief SetObjectParent: Changes the parent of an object
+     *
+     * @param object Object name in CoppeliaSim
+     * @param parent Name of the new Parent object
+     * @param keepInPlace When true, the object pose follows the parent object
+     */
+    ds.make_call("CoppeliaSim::SetObjectParent",
```

Comment:
For usability, I think we should have:
- `CoppeliaSim::AttachObject(const std::string & object, const std::string & parent)`
- `CoppeliaSim::DetachObject(const std::string & object)`

Since there's no way to ensure `parent` can give a `-1` id required to detach the object